components/molecules/AuthSection/auth-section.tsx

```diff
@@ -117,7 +117,16 @@ const AuthSection: React.FC = ({}) => {
                   <Divider type="vertical" className="!h-6 !bg-light-slate-6"></Divider>
                 </>
               )}
-              <button className="relative cursor-pointer" onClick={() => router.push(`/user/notifications`)}>
+              <button
+                key="notifications"
```

Comment:
Refactor (blocking): The `key` prop is only required for items in an array.

Reply:
Done.